sdk/core/src/context.rs

```diff
@@ -19,9 +32,23 @@ impl Context {
     pub fn new() -> Self {
         Self {
             type_map: HashMap::new(),
+            previous_context: None,
         }
     }
 
+    /// Creates a new, empty context that wraps the previous context.
+    pub fn from_previous_context(context: impl Into<Arc<Context>>) -> Self {
```

Comment:
Not sure if I love the name `from_previous_context`, but not sure of a good name

Reply:
I'll do the change detailed above and see if the name "presents" itself then 🕐!

Reply:
As expected, the solution presented itself 👍. Now the `OverridableContext` is constructed by calling `create_override` directly from the `Context`.